packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -334,17 +328,19 @@ export default class extends DataAggregatingRouteHandler {
     const {
       id,
       dataElementCode,
-      isDataRegional,
       measureBuilderConfig,
       measureBuilder,
+      data_services: dataServices,
     } = mapOverlay;
 
     const entityCode = shouldFetchSiblings
       ? await this.getCountryLevelOrgUnitCode()
       : this.entity.code;
     const entity = await this.models.entity.findOne({ code: entityCode });
-    const dataServices = createDataServices(mapOverlay);
-    const dhisApi = getDhisApiInstance({ entityCode: this.entity.code, isDataRegional });
+    const dhisApi = getDhisApiInstance({
+      entityCode: this.entity.code,
+      isDataRegional: dataServices?.[0]?.isDataRegional,
```

Comment:
That optional chaining for arrays came up again soon!

Reply:
Yeah was a time saver having just gone through that review or I guarantee I would've got it wrong